### DISCUSSION TITLE
Proper way to customize error pages

### DISCUSSION THREAD

**Discussion by code-monad** [rwf2/Rocket#2785] [Questions] [answered]
So the Rocket will have it's error page like this:
![image](https://github.com/rwf2/Rocket/assets/45656526/a05d1c15-0bb6-486d-8dbc-bce38e33dcfc)

Is there anyway i can customize the style of it? Or how to?

**Comment by wezm** [comment, marked as answer]
Check out error catchers: https://rocket.rs/guide/v0.5/requests/#error-catchers

**Reply by code-monad** [reply]
thx! helps alot
